website/common/script/ops/buy/abstractBuyOperation.js

```diff
@@ -0,0 +1,133 @@
+import i18n from '../../i18n';
+import {
+  NotAuthorized,
+} from '../../libs/errors';
+import _merge from 'lodash/merge';
+import _get from 'lodash/get';
+
+export class NotImplementedError extends Error {
```

Comment:
should this be extracted to somewhere more generic since it'll be likely used in other classes as well?

Reply:
will do